### PR TITLE
[WINESYNC][USER32][WORDPAD] Use GetSysColorBrush()

### PR DESCRIPTION
**Pull request by SergeGautherie**
Use the cached object.

Kind of follow-up to be01412 (0.4.15-dev-5248).

### CODE REVIEW

File: base/applications/wordpad/print.c [hunk 1]
```diff
@@ -381,10 +381,9 @@ static void paint_ruler(HWND hWnd, LONG EditLeftmost, BOOL NewMetrics)
     HDC hdcPrint = make_dc();
     RECT printRect = get_print_rect(hdcPrint);
     RECT drawRect;
-    HBRUSH hBrush = CreateSolidBrush(GetSysColor(COLOR_MENU));
 
     GetClientRect(hWnd, &drawRect);
-    FillRect(hdc, &drawRect, hBrush);
+    FillRect(hdc, &drawRect, GetSysColorBrush(COLOR_MENU));
```
Comment (binarymaster) [inline base/applications/wordpad/print.c:386]: This is Wine code: https://github.com/reactos/reactos/blob/a17664cec2ca0f98857e47b05d11cf5992096206/media/doc/WINESYNC.txt#L252

File: win32ss/user/user32/windows/mdi.c [hunk 2]
```diff
@@ -931,10 +930,8 @@ static BOOL MDI_AugmentFrameMenu( HWND frame, HWND hChild )
         hBitmap = CreateCompatibleBitmap(hdc, cx, cy);
         hOldBitmap = SelectObject(hMemDC, hBitmap);
         SetMapMode(hMemDC, MM_TEXT);
-        hBrush = CreateSolidBrush(GetSysColor(COLOR_MENU));
-        DrawIconEx(hMemDC, 0, 0, hIcon, cx, cy, 0, hBrush, DI_NORMAL);
+        DrawIconEx(hMemDC, 0, 0, hIcon, cx, cy, 0, GetSysColorBrush(COLOR_MENU), DI_NORMAL);
         SelectObject (hMemDC, hOldBitmap);
-        DeleteObject(hBrush);
```
Comment (binarymaster) [inline win32ss/user/user32/windows/mdi.c:937]: This is Wine code as well: https://github.com/reactos/reactos/blob/a17664cec2ca0f98857e47b05d11cf5992096206/media/doc/WINESYNC.txt#L344

**Review by HBelusca — COMMENTED** [review]
For a minimal number of changes, only the two `DeleteObject(hBrush);` calls could be removed, and the two `CreateSolidBrush(GetSysColor(...))`  calls be replaced by `GetSysColorBrush`, while keeping the intermediate `hBrush` variable.

File: base/applications/wordpad/print.c (hunk 1 above)
Comment (SergeGautherie) [reply base/applications/wordpad/print.c:386]: Yeah, that's what I wrote.

File: win32ss/user/user32/windows/mdi.c (hunk 2 above)
Comment (SergeGautherie) [reply win32ss/user/user32/windows/mdi.c:937]: Indeed. My bad, I got fooled by recent changes without `__REACTOS__`...

**Review by binarymaster — APPROVED** [review]
LGTM, but a few things to make it perfect:
```
git commit --amend --author="Zebediah Figura <z.figura12@gmail.com>"
```
^ for the both commits, and then you can simplify commit messages:
```patch
 [WINESYNC][USER32] MDI_AugmentFrameMenu(): Use GetSysColorBrush()
 
-WineSync applicable part of wine-4.7 Zebediah Figura's
-https://source.winehq.org/git/wine.git/commitdiff/8d251a1dd1d523bd20c30b355b79421340f53a80
+Import applicable part of commit from Wine 4.7:
+https://source.winehq.org/git/wine.git/commit/8d251a1dd1d523bd20c30b355b79421340f53a80
```
```patch
 [WINESYNC][WORDPAD] paint_ruler(): Use GetSysColorBrush()
 
-WineSync wine-4.7 Zebediah Figura's
+Import commit from Wine 4.7:
 https://source.winehq.org/git/wine.git/commit/dcd9376ae711d56778032f146480e1d1aaf245ef
```

**Comment by binarymaster** [comment]
Merged manually in dee3e6e7cdc.

**Pull request closed without merging**